RetailCoder.VBE/Inspections/QuickFixes/PassParameterByReferenceQuickFix.cs

```diff
@@ -15,11 +15,13 @@ namespace Rubberduck.Inspections.QuickFixes
     public class PassParameterByReferenceQuickFix : QuickFixBase
     {
         private Declaration _target;
+        private AssignedByValParameterQuickFixHelper _quickFixHlpr;
```

Comment:
In the follow-on to this PR (PR #2733) I elected to keep the PassParameterByRefernceQuickFIx named as-is.  Even though it is currently only used by the AssignedByValParameter inspection, it looks as though it would be useful for any inspection that has a need to change a variable from ByVal to ByRef.